api/instance/v1/volume_utils_test.go

```diff
@@ -0,0 +1,74 @@
+package instance
+
+import (
+	"testing"
+
+	"github.com/scaleway/scaleway-sdk-go/internal/testhelpers"
+	"github.com/scaleway/scaleway-sdk-go/internal/testhelpers/httprecorder"
+	"github.com/scaleway/scaleway-sdk-go/utils"
+)
+
+func TestUpdateVolmue(t *testing.T) {
+
+	client, r, err := httprecorder.CreateRecordedScwClient("volume-utils-test")
+	testhelpers.Ok(t, err)
+	defer func() {
+		testhelpers.Ok(t, r.Stop()) // Make sure recorder is stopped once done with it
+	}()
+
+	instanceAPI := NewAPI(client)
+
+	var (
+		zone                 = utils.ZoneFrPar1
+		organization         = "d429f6a1-c0a6-48cf-8b5a-1f9dfe76ffd3"
+		volumeName           = "test volume"
+		volumeSize    uint64 = 20000000000
+		volumeType           = VolumeTypeLSSD
+		newVolumeName        = "some new volume name"
+
+		volumeID string
+	)
+
+	t.Run("Setup: create volume", func(t *testing.T) {
```

Comment:
A `Run` should represent a complete test not just part of one.

Either group everyting in one test or create a inner setup and teardown func that you can call from run